tsconfig.json

```diff
@@ -4,8 +4,8 @@
         "module": "es6",
         "declaration": true,
         "outDir": "./dist",
-        "strict": true
-
+        "strict": true,
```

Comment:
Had a read up on this, seems good.
However once you put this in you have to take responsibility for making sure that all classes have a default initialised value :).